upup/pkg/fi/nodeup/nodetasks/package.go

```diff
@@ -273,10 +273,18 @@ func (_ *Package) RenderLocal(t *local.LocalTarget, a, e, changes *Package) erro
 				return fmt.Errorf("error creating directories %q: %v", localPackageDir, err)
 			}
 
+			// Append file extension for local files
+			var ext string
+			if t.HasTag(tags.TagOSFamilyDebian) {
```

Comment:
the surrounding code here clearly targets linux (apparently debian and redhat derivatives). while i don't expect to see fully built out support right now for anything else, if someone somehow ended up here with an unsupported OS, it might be useful to log an error. something like: `OSFamilyBeOS) nodes are currently unsupported.` 

FYI: i could approve this patch without changing this, but I was interested to hear if that would be useful .

Reply:
I think it's best to add it here also. Initially was thinking it should be handled by the similar checks further down, but you never know what future code changes will look like.
Added more general message "unsupported package system", with tags system is a bit hard to print something better.

I think in the future the tar.gz package may get more attention and this will make it easier to support other distros.
I wouldn't got so far as BeOS, though :)

Reply:
I don't actually want to do anything with BeOS ;) I was just trying to posit a case that likely wasn't going to incite any flames for os-selection. Because it was obviously ridiculous. i'm blaming nostalgia for BeOS references.

Reply:
I know. Just joking ;)